pkg/storage/s3/s3_test.go

```diff
@@ -1239,7 +1231,8 @@ func TestNegativeCasesObjectsStorage(t *testing.T) {
 }
 
 func TestS3Dedupe(t *testing.T) {
-	skipIt(t)
+	tskip.SkipS3(t)
```

Comment:
this is the only place where we need both, the rest should be just S3. they are using boltdb